app/src/main/java/org/gdg/frisbee/android/common/GdgActivity.java

```diff
@@ -282,16 +274,29 @@ public void onConnectionSuspended(int i) {
     @Override
     public void onConnectionFailed(@NonNull ConnectionResult result) {
         if (mSignInProgress != STATE_IN_PROGRESS) {
+            if (!GoogleApiAvailability.getInstance().isUserResolvableError(result.getErrorCode())) {
+                if (!PrefUtils.shouldShowFatalPlayServiceMessage(this)) {
+                    return;
+                }
+                PrefUtils.setFatalPlayServiceMessageShown(this);
```

Comment:
fatal play service messages are those that can't be recoved by the user. I should rename the method